CHANGELOG.rst

```diff
@@ -0,0 +1,345 @@
+Changelog
+=========
+
+Current development version
+---------------------------
+
+.. _v1110:
```

Comment:
I'm worried about these hyperlinks. It seems like `v11.1.0` and `v1.11.0` and `v1.1.10` might collide. Thoughts?

Reply:
There are no hyperlinks to these titles so I am not sure why the autoconverter marked them as explicit hyperlink titles. I removed them. Also, there seems to be a bug in the autoconverter since _v.11.1.0 is a valid reference name.